commands/wiki.js

```diff
@@ -1,5 +1,128 @@
 const { SlashCommandBuilder } = require('discord.js');
 
+const wikiTable = {
+	"home": {
+		name: "Home",
+		url: "https://github.com/official-stockfish/Stockfish/wiki",
+		description: `Stockfish is a free, powerful and open-source UCI chess engine derived from Glaurung 2.1.
```

Comment:
do you think theres an easy way to show the text preview of the page ? I suppose there isnt...

Reply:
Not really, since the pages are of different types. You have to work hard to extract some "preview" out of them.